test-app/package.json

```diff
@@ -112,6 +112,7 @@
     "prettier-plugin-ember-template-tag": "^0.3.0",
     "qunit": "^2.19.3",
     "qunit-dom": "^2.0.0",
+    "sinon": "^15.0.1",
```

Comment:
We shouldn't need this anymore which should remove it from the lock file as well! 🎉